### DISCUSSION TITLE
Markdown Viewer: All links are permanently underlined

### DISCUSSION THREAD

**Discussion by xamidi** [community/community#72172] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I noticed that very recently (in the last few days), there has been a style change in the .md viewer, where all links are now permanently underlined, not anymore only when hovering with a cursor over a link.

I wonder (and hope) this is a bug, because it looks ugly — it makes Markdown files with many links look like web pages in the 90s.

**Comment by rmacklin** [comment, marked as answer]
That can be disabled in your accessibility settings https://github.com/settings/accessibility

![image](https://github.com/community/community/assets/1863540/66ad738b-b8d7-415a-9b4a-d9116ce94b76)

**Reply by xamidi** [reply]
Thank you @rmacklin. At least now only most visitors that don't know of this new 'feature' (or are not logged in) must throw up when they visit my main project on GitHub. IMO, it should be disabled by default since only very few people (1 in 33000) are entirely color blind.
